### PR TITLE
Python3 support. Fixes #9

### LINKED ISSUE #9: python3 support
Fails to build with python3.  2to3 works.  Not sure if it's output is backwards compatible.

### PR DESCRIPTION
**Pull request by graphicore**
I tried to run the tests but I get a lot of noisy output, see #14 I think I'm probably calling the test in a wrong way. Besides that, running the tests on current HEAD with python2 and this branch  bde64f3 with python 2 and 3 seems to produce the same results.

### CODE REVIEW

**Comment by divyekapoor** [comment]
Confirmed that these fixes should add python 3 support. Can we please check these into master and have a new package published to PyPI?

**Comment by gpshead** [comment]
google-apputils users should migrate to absl-py:

https://github.com/abseil/abseil-py & https://pypi.python.org/pypi/absl-py/

It is the modern version of these libraries, officially supporting Python 3.

If it is missing anything (I've heard one call for appcommands), please file a feature request over there.

File: google/apputils/basetest.py [hunk 1]
```diff
@@ -51,6 +55,19 @@
 
 FLAGS = flags.FLAGS
 
+
+# Sadly there is no types.IntTypes defined for us and types.StringTypes is
+# not supported by Python 3.
+if sys.version_info[0] < 3:
+  _INT_TYPES = (int, long)
+  _STRING_TYPES = (basestring)
```
Comment (basak) [inline google/apputils/basetest.py:63]: The brackets here do nothing. If you want a tuple, then use `(basestring,)`.

File: google/apputils/basetest.py [hunk 2]
```diff
@@ -51,6 +55,19 @@
...
+  _UNICODE_TYPE = unicode
+else:
+  _INT_TYPES = (int)
```
Comment (basak) [inline google/apputils/basetest.py:66]: The brackets here do nothing. If you want a tuple, then use `(basestring,)`.

File: google/apputils/basetest.py [hunk 3]
```diff
@@ -51,6 +55,19 @@
...
+  _STRING_TYPES = (str)
```
Comment (basak) [inline google/apputils/basetest.py:67]: The brackets here do nothing. If you want a tuple, then use `(basestring,)`.

**Comment by gpshead** [comment]
i'm closing this pull requests as google-apputils is deprecated, the replacement is absl-py per my earlier comment.  Please work with the maintainers of things that declare a dependency on google-apputils to move over to absl-py.

**Pull request closed without merging**